packages/foam-core/src/markdown-provider.ts

```diff
@@ -422,7 +436,14 @@ export function createMarkdownReferences(
         : dropExtension(relativePath);
 
       // [wiki-link-text]: path/to/file.md "Page title"
-      return { label: link.slug, url: pathToNote, title: target.title };
+      return {
+        label:
+          link.rawText.indexOf('[[') > -1
```

Comment:
labels should only be created for wikilinks, this test should always be true, right?